spec/controllers/books_controller_spec.rb

```diff
@@ -54,7 +54,7 @@
     let(:invalid_book_attributes) { attributes_for(:book, title: '') }
 
     context "when user is signed in and is an admin" do
-      let(:user) { create(:user, admin: true) }
+      let(:user) { create(:user, admin?: true) }
```

Comment:
why this? it should be 'admin' and when checking the value you can use user.admin?